src/agent/coverage/examples/coverage.rs

```diff
@@ -94,6 +94,16 @@ fn input_command(argv: &[String], input: &Path) -> Command {
     cmd
 }
 
+#[cfg(target_os = "macos")]
+fn record(
+    _cache: &mut ModuleCache,
+    _filter: CmdFilter,
+    _cmd: Command,
+    _timeout: Duration,
+) -> Result<Coverage> {
+    unimplemented!("coverage recording is not supported on osx");
```

Comment:
```suggestion
    unimplemented!("coverage recording is not supported on macOS");
```
We should use this naming throughout.